understanding/20/link-purpose-in-context.html

```diff
@@ -1,403 +1,187 @@
+         continuity between the link 'clicked on' and the web page that the user lands on.</p>
       <p>In some situations, authors may want to provide part of the description of the link
          in logically related text that provides the context for the link. In this case the
          user should be able to identify the purpose of the link without moving focus from
          the link. In other words, they can arrive on a link and find out more about it without
          losing their place. This can be achieved by putting the description of the link in
-         the same sentence, paragraph, list item, or table cell as the link, or in the table header cell for a link in a data table, because these are directly associated with the link itself. Alternatively, authors may choose to use an ARIA technique to associate additional
-         text on the page with the link.
-      </p>
-      
+         the same sentence, paragraph, list item, or table cell as the link, or in the table header cell for a link in a
+         data table, because these are directly associated with the link itself. Alternatively, authors may choose to
+         use an ARIA technique to associate additional text on the page with the link.</p>
       <p>This context will be most usable if it precedes the link. (For instance, if you must
          use ambiguous link text, it is better to put it at the end of the sentence that describes
          its destination, rather than putting the ambiguous phrase at the beginning of the
          sentence.) If the description follows the link, there can be confusion and difficulty
-         for screen reader users who are reading through the page in order (top to bottom).
-         
-      </p>
-      
+         for screen reader users who are reading through the page in order (top to bottom).</p>
       <p>It is a best practice for links with the same destination to have consistent text
-         (and this is a requirement per 
-         <a href="consistent-identification" class="guideline">Success Criterion 3.2.4</a> for pages in a set). It is also a best practice for links with different purposes
-         and destinations to have different link text.
-      </p>
-      
-     <p>A best practice for links to <a>conforming alternate versions</a> is to ensure that the link text to the conforming alternate version indicates in link text that the page it leads to represents the more accessible version. This information may also be provided in text - the goal is to ensure that the end user knows what the purpose of the link is. </p>
-     
+         (and this is a requirement per
+         <a href="consistent-identification" class="guideline">Success Criterion 3.2.4</a> for pages in a set). It is
+         also a best practice for links with different purposes
+         and destinations to have different link text.</p>
+      <p>A best practice for links to <a>conforming alternate versions</a> is to ensure that the link text to the
```

Comment:
I know my suggestion would make the Understanding text even longer, but should there be something about the inverse case (links with different text leading to the same target)? I seem to remember that some people have been seen failing content just for that...

Reply:
This is already mentioned/covered by the "best practice for links with the same destination to have consistent text" ... as that is best practice, it follows (for me at least) that it's NOT a failure if links with different text point to the same destination?

Reply:
I wasn't suggesting this should fail, a note should rather serve to explain that this is not a failure. And if already covered, fine, I didn't spot that.

Reply:
I'd say yes, it's implicit in the fact that this says it's a best practice. i.e. not a requirement, so not doing that won't necessarily fail anything.